addons/website/static/src/snippets/s_showcase/options.js

```diff
@@ -12,7 +12,7 @@ options.registry.Showcase = options.Class.extend({
         const isLeftCol = $showcaseCol.index() <= 0;
         const $title = this.$target.children('.s_showcase_title');
         $title.toggleClass('flex-lg-row-reverse', isLeftCol);
-        $title.find('.s_showcase_icon').toggleClass('mr-lg-0 ml-3', isLeftCol);
+        $title.find('.s_showcase_icon').toggleClass('mr-lg-0 ml-lg-3', isLeftCol);
```

Comment:
People using 14.0 right now may have their JS updated but not their views, so this has to work with old views (and actually old snippets) too. I guess you have to remove the ml-3 class each time too for compatibility.

Reply:
I replace the ml-3 class in all column's blocks to keep them aligned (to avoid  having a mix of ml-3 and ml-lg-3)